server/services/database/profile.js

```diff
@@ -0,0 +1,21 @@
+const { Pool } = require("pg");
+const config = require("../../config");
+const pool = new Pool(config);
+
+const createProfile = (profile) => {
```

Comment:
You can destructuring these params, e.g `const createProfile = ({ job_title, bio, location, employment_status }) =>`